### PR TITLE
added the files for payment-api-prod

### PR DESCRIPTION
**Pull request by AnilDhawaleGitHub**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: k8s/prod/common/ccpay/ccpay-payment-app.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    java:
+      replicas: 2
+      image: hmctspublic.azurecr.io/payment/api:prod-a4872e81
+      ingressHost: payment-api-prod.service.core-compute-prod.internal
+      applicationInsightsInstrumentKey: "f0e045d3-c9e4-49bc-87da-c63f05e82cfa"
+      environment:
+        SPRING_DATASOURCE_URL: "jdbc:postgresql://payment-postgres-db-prod.postgres.database.azure.com:5432/payment"
+        SPRING_DATASOURCE_USERNAME: payment@payment-postgres-db-prod
+        AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
```
Comment (luigibk) [inline k8s/prod/common/ccpay/ccpay-payment-app.yaml]: You shouldn't need this

File: k8s/prod/common/ccpay/ccpay-payment-app.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          - pci-pal-api-key
+          - gov-pay-keys-reference
+          - gov-pay-keys-cmc
+          - gov-pay-keys-divorce
+          - gov-pay-keys-probate
+          - liberata-keys-oauth2-client-id
+          - liberata-keys-oauth2-client-secret
+          - liberata-keys-oauth2-username
+          - liberata-keys-oauth2-password
+        enabled: true
```
Comment (luigibk) [inline k8s/prod/common/ccpay/ccpay-payment-app.yaml]: Why are the sercrets repeated here?

File: k8s/prod/common/ccpay/ccpay-payment-app.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      replicas: 2
+      image: hmctspublic.azurecr.io/payment/api:prod-a4872e81
+      ingressHost: payment-api-prod.service.core-compute-prod.internal
+      applicationInsightsInstrumentKey: "f0e045d3-c9e4-49bc-87da-c63f05e82cfa"
+      environment:
+         AUTH_IDAM_CLIENT_BASEURL: https://idam-api.platform.hmcts.net
+      aadIdentityName: ccpay
+    global:
+      environment: prod
+      subscriptionId: "8999dec3-0104-4a27-94ee-6588559729d1"
```
Comment (timja) [inline k8s/prod/common/ccpay/ccpay-payment-app.yaml]: not needed
```suggestion
```

File: k8s/prod/common/ccpay/ccpay-payment-app.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (timja) [inline k8s/prod/common/ccpay/ccpay-payment-app.yaml]: missing CODEOWNERS file changes

Comment (AnilDhawaleGitHub) [reply k8s/prod/common/ccpay/ccpay-payment-app.yaml]: its done

**Pull request merged**